src/riak_kv_cinfo.erl

```diff
@@ -54,7 +54,7 @@ aae_throttle(CPid) ->
                         [ThrottleLimits]).
 
 get_aae_throttle() ->
-    [{throttle_kill, riak_kv_entropy_manager:get_aae_throttle_kill()},
+    [{throttle_enabled, riak_kv_entropy_manager:is_aae_throttle_enabled()},
```

Comment:
This value seems to be thrown away in all cases I've found (riak_test and riak proper, the function, while exported, is only called from this module). Can we check to see if we can just drop the throttle_enabled key/value in this proplist altogether please?